src/defaultV2-theme/variables.scss

```diff
@@ -62,71 +58,98 @@ $font-size: var(--sjs-font-size, 16px);
 
 $font-surveytitle-family: var(--sjs-font-surveytitle-family, var(--font-family));
 $font-surveytitle-weight: var(--sjs-font-surveytitle-weight, 700);
-$font-surveytitle-size: var(--sjs-font-surveytitle-size, calc(2 * var(--sjs-font-size, $font-size)));
+$font-surveytitle-size: var(--sjs-font-surveytitle-size, calc(2 * ($font-size)));
```

Comment:
We can't simply remove --sjs-font-size here - tile font will stop scaling on --sjs-font-size change